.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
```

Comment:
@seiwan This should be removed and added to your default .gitignore

Reply:
File deleted